src/store/address/constants.ts

```diff
@@ -11,7 +11,7 @@ export const REQUIRED_FIELDS: AddressRequirements = {
 	[ AddressTypeModel.COMPANY_WITH_CONTACT ]: [ 'salutation', 'firstName', 'lastName', 'companyName', 'street', 'postcode', 'city', 'country', 'email', 'addressType' ],
 	[ AddressTypeModel.EMAIL ]: [ 'email', 'salutation', 'firstName', 'lastName' ],
 	[ AddressTypeModel.ANON ]: [],
-	[ AddressTypeModel.UNSET ]: [ 'addressType' ],
+	[ AddressTypeModel.UNSET ]: [ 'salutation', 'firstName', 'lastName', 'street', 'postcode', 'city', 'country', 'email', 'addressType' ],
```

Comment:
I was confused by this line, why are these fields required when the address type is unset? I guess it's to cover the case you described in the comment, when the address type is unset, but should behave like "Person" because those are the fields that are displayed? If that's the case then please put a comment here to make it less confusing.